### DISCUSSION TITLE
Editing ALT tags

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#6478] [General]
Hi

There's no way to know if an image has alt text unless we click into each image.

- Could list view in assets be updated to show if an image has an alt tag or not? 
- Perhaps an icon could show if the image is missing alt text?
- The ability to filter images if they don't have an alt tag would be useful too
- Or maybe alt tags could be viewed in list mode? If so, could they be added/edited in list mode. Avoiding the need to click into the image?

I only ask because I've got a client who's added lots of images and hasn't added any alt text! I only found this out as part of a site scan I did with Ahrefs.

**Comment by QuentinGab** [comment]
The Peak starter kit https://github.com/studio1902/statamic-peak implement a widget to display images without alt text in the dashboard, I'm pretty sure you can reuse it easily in your site
